pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -2577,6 +2577,27 @@ substring  true         false      i            NULL
 substring  true         false      i            NULL
 substring  true         false      i            NULL
 
+# Verify that proretset is true for set-returning functions.
+query OTBT
+SELECT oid, proname, proretset, prorettype FROM pg_catalog.pg_proc
+WHERE proname IN (
+  'jsonb_object_keys', 'generate_series', 'generate_subscripts',
+  'json_populate_recordset', 'unnest', 'json_each'
+) ORDER BY 1
+----
+317  generate_series          true  20
```

Comment:
are these oids deterministic?

Reply:
yeah they are for builtins, thanks to: https://github.com/cockroachdb/cockroach/blob/9f61a38b670e57873509568f67bb665e50cd1935/pkg/sql/sem/builtins/fixed_oids.go#L15-L27

Reply:
oh pffft right; ty